ios/Classes/Blemulator/Model/Service.m

```diff
@@ -0,0 +1,25 @@
+#import "Service.h"
+
+@implementation Service
```

Comment:
Was it not possible to reuse the class from MBA or CoreBluetooth?

Reply:
It's a part of RxBluetoothKit and it's not exposed to objc (would require some changes in RxBluetoothKit)